sidebars.js

```diff
@@ -2,50 +2,64 @@ module.exports = {
   docs: [
     {
       type: 'doc',
-      id: 'getting-started',
-    },
-    {
-      type: 'doc',
-      id: 'payid-cli',
+      id: 'welcome-to-payid',
     },
     {
       type: 'category',
-      label: 'PayID Protocol',
+      label: 'Reference Implementation',
       items: [
-        'payid-overview',
-        'payid-headers',
-        'payid-interfaces',
-        'payid-schemas',
+        'payid-reference-overview',
+        'payid-server-requirements',
+        'payid-cli',
+        'verifiable-payid',
+        'payid-best-practices',
+        'integrate-payid-users',
+        'metrics',
       ],
     },
     {
-      type: 'doc',
-      id: 'verifiable-payid',
+      type: 'category',
+      label: 'Deployment Options',
+      items: [
+        'intro-deploy',
+        'local-deployment',
+        'deploy-with-docker',
+        'aws-lambda-deploy',
+        'remote-deployment',
+      ],
     },
     {
       type: 'category',
-      label: 'Reference Implementation',
+      label: 'Get Started',
       items: [
-        'payid-reference-overview',
-        'api-reference',
-        'payid-best-practices',
-        'integrate-payid-users',
-        'metrics',
+        'getting-started',
+        'payid-setup-tutorial',
+        'payid-cli-tutorial',
+        'verifiable-payid-tutorial',
+        'xpring-sdk-payid-tutorial',
+        'metrics-tutorial',
       ],
     },
```

Comment:
As discussed, I suggest moving this up to the second position in the left nav.

Reply:
Done.